<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocStatusClassItemEditorDialog</class>
 <widget class="QDialog" name="SocStatusClassItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="6" column="1" colspan="4">
    <widget class="QCheckBox" name="chkAutoDateClose">
     <property name="text">
      <string>Закрывать старую запись соц.статуса данного класса &quot;вчерашней датой&quot;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLineEdit" name="edtCode"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>На&amp;именование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblTypes">
     <property name="text">
      <string>Льготы</string>
     </property>
     <property name="buddy">
      <cstring>tblTypes</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="edtName">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>73</width>
       <height>171</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblCode">
     <property name="text">
      <string>&amp;Код</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="chkTightControl">
     <property name="text">
      <string>Жёсткий контроль</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" rowspan="2" colspan="4">
    <widget class="CInDocTableView" name="tblTypes"/>
   </item>
   <item row="2" column="4">
    <widget class="QCheckBox" name="chkShowInClientInfo">
     <property name="text">
      <string>Отображать в шилде</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="chkSoftControl">
     <property name="text">
      <string>Мягкий контроль</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>tblTypes</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SocStatusClassItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SocStatusClassItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
